services/app-api/prisma/migrations/20220208221835_add_revisions_table/migration.sql

```diff
@@ -0,0 +1,19 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+-- CreateTable
+CREATE TABLE "StateSubmissionRevision" (
+    "id" TEXT NOT NULL,
+    "createdAt" TIMESTAMP(3) NOT NULL,
+    "submissionID" TEXT NOT NULL,
+    "submissionFormProto" BYTEA NOT NULL,
+
+    CONSTRAINT "StateSubmissionRevision_pkey" PRIMARY KEY ("id")
+);
+
+-- AddForeignKey
+ALTER TABLE "StateSubmissionRevision" ADD CONSTRAINT "StateSubmissionRevision_submissionID_fkey" FOREIGN KEY ("submissionID") REFERENCES "StateSubmission"("id") ON DELETE RESTRICT ON UPDATE CASCADE;
```

Comment:
what the best way to verify the relationship between the StateSubmission and StateSubmissionRevision is set up properly? I'm guessing by making a SQL query so trying that